homeassistant/components/xiaomi_ble/sensor.py

```diff
@@ -29,10 +29,13 @@
     TEMP_CELSIUS,
 )
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers.bluetooth_device import (
```

Comment:
I think `bluetooth` is sufficient in the helper name.

Reply:
The underlying library that this is for isn't strictly a Bluetooth library. I think it ultimately leads here?

https://github.com/Bluetooth-Devices/sensor-state-data/blob/main/src/sensor_state_data/data.py#L23

So maybe the helper shouldn't marked as Bluetooth specific at all.

Reply:
@Jc2k The library is currently used only with Bluetooth devices, and is hosted under the Bluetooth-Devices org. If it becomes more generic at some point, the helper could be renamed then..?

Reply:
Regardless of where it lives on GH, the library the device class is from has nothing to do with Bluetooth, it was designed to not contain any Bluetooth code. That's why there is also the sister package bluetooth-sensor-state-data. Your helper is for a class in the generic non-Bluetooth pypi project.

The repo also contains a dbus library, if this was a dbus helper I wouldn't expect it to be under the Bluetooth namespace in HA because it was maintained by HA Bluetooth people.

Reply:
The libraries are split into `bluetooth-sensor-state-data` and `sensor-state-data` for this very reason.

I agree with @Jc2k as I expect I will want to use `sensor-state-data` for non-bluetooth integrations in the future and it would be weird to have it under a bluetooth namespace in Home Assistant